app/screens/edit_post/edit_post.js

```diff
@@ -153,8 +159,23 @@ export default class EditPost extends PureComponent {
             this.setState({message});
         }
 
+        const tooLong = this.isMessageTooLong(message);
+        if (tooLong) {
+            const errorLine = this.context.intl.formatMessage({
+                id: 'mobile.message_length.message_multiline',
```

Comment:
That original string wrapped ugly into two because it didn't fit, so I created this new multiline one that split into two and looked (a little bit) better.